javascript/ql/src/semmle/javascript/PackageExports.qll

```diff
@@ -68,4 +68,9 @@ private DataFlow::Node getAnExportFromModule(Module mod) {
   result.analyze().getAValue() = mod.(NodeModule).getAModuleExportsValue()
   or
   exists(ASTNode export | result.getEnclosingExpr() = export | mod.exports(_, export))
+  or
```

Comment:
The problem is that `Module::exports` just exposes an `ASTNode` and there is no reliable way to map that to the correct data-flow node. It would be great to modernize that abstraction but I don't think this PR is the right place to do that.